### DISCUSSION TITLE
Set Icon of Custom OAuth-Provider

### DISCUSSION THREAD

**Discussion by RedstonePfalz** [zitadel/zitadel#7232] [Q&A] [answered]
Hallo,
Is there a way to show an icon with the custom OAuth provider like there is with GitLab, GitHub and Google?
I hope someone can help me.

**Comment by hifabienne** [comment, marked as answer]
Hi @RedstonePfalz 
At the moment there is no possibility to upload a icon for a generic idp.
We do already have an issue in our backlog.
If you are interested in it, please add a reaction, so we can see how many customers do want to have the feature.

https://github.com/zitadel/zitadel/issues/5455
